open-metadata-implementation/access-services/subject-area/subject-area-api/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/properties/objects/graph/Graph.java

```diff
@@ -120,9 +120,14 @@ public String toString(StringBuilder sb) {
         if (sb == null) {
             sb = new StringBuilder();
         }
-        sb.append("NodeGUID=" + rootNodeGuid + "\n");
-        sb.append(("NodeTypeFilter=" + nodeFilter == null || nodeFilter == "") ? "none" : nodeFilter + "\n");
-        sb.append(("relationshipTypeFilter=" + relationshipFilter == null || relationshipFilter == "") ? "none" : relationshipFilter + "\n");
+        /*
+        The below code is triggering alerts in a number of scanners
+        It's not clear why only the first string + raised an alert and the two remaining calls didn't ?
+        using Objects.equals was recommended as being null-safe
+         */
+        sb.append("NodeGUID=").append(rootNodeGuid).append("\n");
+        sb.append((Objects.equals(nodeFilter, "")) ? "none" : nodeFilter + "\n");
```

Comment:
I assume you made the line 128 change to add in the append rather than using the + operator. If we are doing that we should change this line and the next to use append as well in the ternary else.

Reply:
@davidradl  this fix wasn't planned , it was thrown up by the QL scan after the PR was opened.
In an attempt to keep the review as simple as possible we've adopted a policy of only changing exactly what the scan is alerting us against, we've tried the "cleaning up while we are there" approach and it quickly became complicated. 
So that's why I didn't make the changes you suggest , but if your prepared to approve it , I'll happily make the changes you suggest.